### PR TITLE
Prepare for release

### PR DESCRIPTION
**Pull request by mgsharm**
*Description of changes:*

* All commits in this PR [link](https://github.com/awslabs/tough/compare/tuftool-v0.11.1...develop). 

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: tough-kms/CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,17 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [0.11.0] - 2024-10-10
+### Changes
+- Fix 128bit integer serialization (thanks @aumetra) [#820]
```
Comment (jpculp) [inline tough-kms/CHANGELOG.md]: I don't think this is relevant to `tough-kms`'s changelog.

File: tough-ssm/CHANGELOG.md [hunk 2]
```diff
@@ -4,6 +4,16 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [0.14.0] - 2024-10-10
+### Changes
+- Fix 128bit integer serialization (thanks @aumetra) [#820]
```
Comment (jpculp) [inline tough-ssm/CHANGELOG.md]: Same as above.

**Review by cbgbt — APPROVED** [review]

File: tough-kms/CHANGELOG.md (hunk 1 above)
Comment (mgsharm) [reply tough-kms/CHANGELOG.md]: Done.

File: tough-ssm/CHANGELOG.md (hunk 2 above)
Comment (mgsharm) [reply tough-ssm/CHANGELOG.md]: Done.

**Review by jmt-lab — APPROVED** [review]

**Pull request merged**